addons/web/static/src/webclient/settings_form_view/settings_form_view.xml

```diff
@@ -6,11 +6,11 @@
         </xpath>
         <xpath expr="//Layout/t[@t-set-slot='layout-actions']" position="replace">
             <t t-set-slot="layout-actions">
-                <div class="o_cp_searchview d-flex flex-grow-1" role="search">
-                    <div class="o_searchview pb-1 align-self-center border-bottom flex-grow-1" role="search" aria-autocomplete="list">
-                        <i class="o_searchview_icon oi oi-search" role="img" aria-label="Search..." title="Search..." />
-                        <div class="o_searchview_input_container">
-                            <input type="text" class="o_searchview_input" accesskey="Q" placeholder="Search..." role="searchbox" t-model="searchState.value" t-ref="autofocus"/>
+                <div class="o_cp_searchview d-flex input-group" role="search">
```

Comment:
This is not a `btn-group` since there is no dropdown in settingsView.
```suggestion
                <div class="o_cp_searchview d-flex w-100" role="search">
```